packages/db/_admin/index.js

```diff
@@ -5,14 +5,34 @@ const Swagger = require('@fastify/swagger')
 module.exports = async (app, opts) => {
   await app.register(Swagger, {
     routePrefix: 'documentation',
-    exposeRoute: true,
```

Comment:
Do we want to support this option? It looks like routes will be exposed anyway.

Reply:
It's implemented a few lines down: https://github.com/platformatic/platformatic/pull/198/files#diff-16401811e8ccdee916c898d3ad1714120fe0e1da342748d7152abbdb60853be2R16

Reply:
There wasn't before for this, it was always set to true. Those are used by the admin dashboard anyway.